.travis.yml

```
@@ -20,6 +20,10 @@ matrix:
       env: DB=MYSQL PDO=1 PHPUNIT_TEST=1
     - php: 7.4
       env: DB=MYSQL BEHAT_TEST=1 BEHAT_SUITE="asset-admin --config vendor/silverstripe/asset-admin/behat.yml"
+    - php: 8.0
+      env: DB=PGSQL PHPUNIT_TEST=1
+    - php: 8.0
+      env: DB=MYSQL PHPUNIT_TEST=1 PHPUNIT_COVERAGE_TEST=1
```

We'll remove this for now until we can figure out what the problem with the segmentation fault is.

```suggestion
    - # php: 8.0
      # env: DB=MYSQL PHPUNIT_TEST=1 PHPUNIT_COVERAGE_TEST=1
```

---

Just update your matrix to what was match what was recently committed - https://github.com/silverstripe/silverstripe-graphql/blob/master/.travis.yml#L12, PHP_COVERAGE_TEST was removed